### PR TITLE
Invoice docs update custom fields tags

### PR DESCRIPTION
**Pull request by reshmabidikar**
Includes the following changes for the invoice custom fields/tags endpoints:

- Corrections to endpoint descriptions/request/response/query parameters
- Corrections to Shell, Java, Ruby, and Python code
- Additional JS/PHP code snippets

### CODE REVIEW

File: source/includes/_invoice.md [hunk 1]
```diff
@@ -2896,13 +3021,13 @@ If successful, returns a status code of 204 and an empty body.
...
 
-See [Account Tags](account.html#account-tags) for an introduction.
+See [Tags](tag.html) for an introduction to tags.
 
 The only `system` tag applicable for an `Invoice` is `WRITTEN_OFF` (`00000000-0000-0000-0000-000000000004`), which as it's name indicates, is used to write off an unpaid invoice, bringing its balance to $0.
 
 ### Add tags to invoice
 
-This API adds one or more tags to an invoice. The tag definitions must already exist.
+This API adds one or more tags to an invoice. The The [tag definition](#tag-definition) corresponding to the tag being added must already exist.
```
Comment (pierre) [inline source/includes/_invoice.md:3030]: Double `The`

File: source/includes/_invoice.md [hunk 2]
```diff
@@ -2641,16 +2646,45 @@ invoice.add_custom_field(custom_field,
...
+$customField -> setName('Test Custom Field');
+$customField -> setValue('test_value');
+$body = array($customField);
+
+$result = $apiInstance->createInvoiceCustomFields($body, $xKillbillCreatedBy, $invoiceId, $xKillbillReason, $xKillbillComment);
+````
+
 **Request Body**
 
 A list of [Custom Field](custom-field.html#custom-field-custom-field-resource) objects. Each object should specify at least the the `name` and `value` attribute. For example:
```
Comment (pierre) [inline source/includes/_invoice.md:2690]: Double `the`

**Review by pierre — APPROVED** [review]
Just typos (`the the`). Otherwise, good to merge!

File: source/includes/_invoice.md (hunk 1 above)
Comment (reshmabidikar) [reply source/includes/_invoice.md]: Fixed

File: source/includes/_invoice.md (hunk 2 above)
Comment (reshmabidikar) [reply source/includes/_invoice.md]: Fixed

**Pull request merged**